src/utils.ts

```diff
@@ -104,6 +104,12 @@ export function getTSConfig(globals, collectCoverage: boolean = false) {
 
     config.inlineSourceMap = true;
     config.inlineSources = true;
+
+    if (config.outDir) {
```

Comment:
- Yes, that `if` can be removed too
- My mistake - it's fine where it is. For some reason, I thought this was in the first `if` block